### PR TITLE
Create ImportCSV.sql

### PR DESCRIPTION
**Pull request by orbruno**
SQL script to import CSV files generated by CreateCSV (dumped from a H2 db) into a MySQL or a MariaDB

### CODE REVIEW

File: H2-migration/ImportCSV.sql [hunk 1]
```diff
@@ -0,0 +1,279 @@
+#Load oc_series_elements
+LOAD DATA INFILE "/Users/orlo/Documents/Current_Projects/ELAN/Opencast/h2/oc_series_elements.csv"
```
Comment (ebbertd) [inline H2-migration/ImportCSV.sql:2]: The path is always hardcoded to your environment. Please change that to a relative path and add a readme on how to use this.

**Comment by ebbertd** [comment]
Furthermore I think this PR should be merged with #32 so that we would have one PR that is structured similar to the other scripts. So a folder, scripts and a readme.

**Comment by lkiesow** [comment]
I don't think this is maintainable. This script relies on internal database structure which changes all the time. There was a recent database change which may already mean that this does not work any longer as expected. If we want to keep something like this around, we need something agnostic to the specific database structure right now.

**Comment by gregorydlogan** [comment]
Agree.  Closing due to both staleness and lack of maintenance.  If you still want this, and are maintaining it we can talk, but until then let's leave this alone.

**Pull request closed without merging**